modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -10440,29 +10440,40 @@ void releaseAccessible() {
 
     /**
      * Tries to move the focus from this {@code Node} in the specified direction.
+     * The {@code Node} serves as a reference point and does not have to be focused or focusable.
      * A successful traversal results in a new {@code Node} being focused.
      * <p>
-     * The {@code visible} parameter specifies whether the {@link #focusVisibleProperty() focusVisible}
-     * flag will be set on the node that receives focus.
-     * Callers must specify {@code true} if this method is called as a result of keyboard navigation,
-     * or if the current node visibly indicates focus;
-     * in all other cases, callers must specify {@code false}.
+     * This method is expected to be called in response to a {@code KeyEvent}, since the {@code Node}
+     * receiving focus will have {@link #focusVisibleProperty() focusVisible} property set.
```

Comment:
I think there should be the definite article "the" in front of `{@link #focusVisibleProperty() focusVisible}`.

Reply:
you are right.  fixed.